app/src/main/res/drawable/ic_warning_black.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
```

Comment:
2 space indentation

Reply:
Oh yes ! I completely forgot to indent these 
New commit soon !